test/e2e/config/docker.yaml

```diff
@@ -110,6 +110,7 @@ intervals:
   default/wait-delete-cluster: ["3m", "10s"]
   default/wait-machine-upgrade: ["20m", "10s"]
   default/wait-machine-pool-upgrade: ["5m", "10s"]
+  default/wait-nodes-ready: ["10m", "10s"]
```

Comment:
I'm not sure about adding a new config option here for this case and how to name it correctly.

Reply:
I'm ok for adding a new config, but soon or later we should audit usage for all those constants and rationalise where possible and document usage for what is left...